Sources/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -110,11 +110,17 @@ extension Graph where Value == Int {
         self.init([source, sink] + internalNodes)
         for node in internalNodes {
             insertEdge(from: source, to: node, value: 1)
+            // FIXME: insertEdge(from: source, to: node, value: epsilon *
```

Comment:
Thanks @bwetherfield! Apologies for my continued illiteracy: is this `lookupTable` supposed to be filled already (for the purpose of initializing the `Graph`), or is this supposed to be done in the process of manipulating an already-initialized `Graph`. What form is this `lookupTable` supposed to take here?

Also, is the `bigM` here arbitrarily large (in our case, `Double.greatestFiniteMagnitude`), or is it parameterized (i.e., customizable by the user)?

Reply:
I'm working on a bigM branch to handle this stuff!

lookupTable should be a big array of real values from which we instantiate the edge weights for our graph (later we need to add scale factors like proximity in score, time in score etc)